### PR TITLE
Extra paths for tags

### PR DESCRIPTION
**Pull request by FRidh**

### CODE REVIEW

File: config.public.json [hunk 1]
```diff
@@ -66,8 +63,83 @@
...
+        "6.topic: printing": [
+            "pkgs/misc/cups",
+        ],
+        "6.topic: python": [
+            "pkgs/top-level/python-packages.nix",
+            "pkgs/development/interpreters/python",
+            "pkgs/development/python-modules",
+            "doc/languages-frameworks/python.md",
+        ],
+        "6.topic: qt/kde": [
```
Comment (grahamc) [inline config.public.json]: cc @ttuegel

File: config.public.json [hunk 2]
```diff
@@ -41,21 +41,18 @@
...
-        "6.topic: python": [
-            "pkgs/top-level/python-packages.nix",
-            "pkgs/development/interpreters/python",
-            "pkgs/development/python-modules",
-            "doc/languages-frameworks/python.md"
-        ],
-        "6.topic: ruby": [
-            "pkgs/development/interpreters/ruby",
-            "pkgs/development/ruby-modules"
+        "6.topic: darwin": [
```
Comment (grahamc) [inline config.public.json]: cc @NixOS/darwin-maintainers

File: config.public.json (hunk 1 above)
Comment (FRidh) [reply config.public.json]: these are taken from the .github/`CODEOWNERS` file

File: config.public.json [hunk 3]
```diff
@@ -41,21 +41,18 @@
...
+            "pkgs/top-level/darwin-packages.nix",
+            "pkgs/stdenv/darwin",
         ],
-        "6.topic: haskell": [
-            "pkgs/development/compilers/ghc",
-            "pkgs/development/tools/haskell",
-            "pkgs/development/haskell-modules",    
-            "pkgs/top-level/haskell-packages.nix"
+        "6.topic: emacs": [
```
Comment (grahamc) [inline config.public.json:48]: this seems good as-is

File: config.public.json [hunk 4]
```diff
@@ -66,8 +63,83 @@
...
+            "pkgs/top-level/haskell-packages.nix",
+            "doc/languages-frameworks/haskell.md",
+        ],
+        "6.topic: kernel": [
+            "pkgs/build-support/linux/kernel",
+        ],
+        "6.topic: nixos": [
+            "nixos",
+        ],
+        "6.topic: policy discussion": [
```
Comment (grahamc) [inline config.public.json]: I'm not sure about this one

File: config.public.json [hunk 5]
```diff
@@ -66,8 +63,83 @@
             "pkgs/development/tools/build-managers/rebar3/",
             "pkgs/development/tools/erlang/",
             "pkgs/development/tools/elixir/",
-            "pkgs/top-level/beam-packages.nix"
-        ]
+            "pkgs/top-level/beam-packages.nix",
```
Comment (grahamc) [inline config.public.json]: Unfortunately JSON doesn't tolerate a `,`  after the last value of a list :(

Comment (FRidh) [reply config.public.json]: Ah, I forgot. Let me remove the trailing comma's.

**Pull request merged**